### PR TITLE
My portfolio

### PR DESCRIPTION
**Pull request by bhassan6621**
my personalized portfolio

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -8,11 +8,35 @@
...
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <div id="header">
+            <h1 id="headerText">BIBI HASSAN</h1>
+            <h2>"Feet on the ground, heart on my sleeves, and head in the clouds." </h2>
```
Comment (ajw029) [inline portfolio/src/main/webapp/index.html]: You can remove the extra new line after "

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -8,11 +8,35 @@
   </head>
   <body>
```
Comment (ajw029) [inline portfolio/src/main/webapp/index.html]: Inside head

I would update the title from "My portfolio" to Bibi's Portfolio. The stuff in the head tag is read by crawlers. I would also add a couple of other metadata tags to optimize SEO performance.

Example:
<meta name="description" content="Anthony Wang Front End Developer Full Stack Portfolio Software Engineer UI UX Design">
<meta name="keywords" content="Anthony Wang, Front, End, Developer, Full Stack, Portfolio, Software Engineer, UI UX Design,CSS, UCSD, Intuit, HTML, Designer, San Diego.">

Some more context:
https://www.wordstream.com/meta-tags
https://www.w3schools.com/tags/tag_meta.asp

File: portfolio/src/main/webapp/index.html [hunk 3]
```diff
@@ -8,11 +8,35 @@
...
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <div id="header">
+            <h1 id="headerText">BIBI HASSAN</h1>
+            <h2>"Feet on the ground, heart on my sleeves, and head in the clouds." </h2>
+            <h2>Click here for a random fact about me:</h2>
+            <button onclick="addRandomGreeting()">Random Fact</button>
+            <div id="greeting-container"></div>
+      </div>
+      <div id=" pictures">
```
Comment (ajw029) [inline portfolio/src/main/webapp/index.html]: Remove the extra space inside of pictures. Actually, i don't see this ID being used anywhere. If its not used anywhere, we can just remove it.

File: portfolio/src/main/webapp/index.html [hunk 4]
```diff
@@ -8,11 +8,35 @@
...
+          <!--photo one -->
+            <img width="300px"  height="300px" src="/images/imageOne.jpg"/>
```
Comment (ajw029) [inline portfolio/src/main/webapp/index.html]: Since all of your images have the same layout, I would recommend creating a class and apply it to the images.

Comment (ajw029) [inline portfolio/src/main/webapp/index.html]: Also, I would add an alt tag to your images. These improve the SEO ranking of your page and it helps people with disabilities (like ppl who use screen readers). 

More context:
https://moz.com/learn/seo/alt-text

File: portfolio/src/main/webapp/index.html [hunk 5]
```diff
@@ -8,11 +8,35 @@
...
+
+         <!--photo two -->
+            <img width="300px"  height="300px" src="/images/imageTwo.jpg"/>
+
+         <!--photo three -->
+            <img width="300px"  height="300px" src="/images/imageThree.jpg"/>
+      </div>
+
+      <div id="pictures_row_2">
+
```
Comment (ajw029) [inline portfolio/src/main/webapp/index.html]: You can remove the extra new line here

File: portfolio/src/main/webapp/index.html [hunk 6]
```diff
@@ -8,11 +8,35 @@
...
+            <img width="300px"  height="300px" src="/images/imageOne.jpg"/>
+
+         <!--photo two -->
+            <img width="300px"  height="300px" src="/images/imageTwo.jpg"/>
+
+         <!--photo three -->
+            <img width="300px"  height="300px" src="/images/imageThree.jpg"/>
+      </div>
+
+      <div id="pictures_row_2">
```
Comment (ajw029) [inline portfolio/src/main/webapp/index.html]: It looks like this ID is not being used anywhere else. We can remove it.

File: portfolio/src/main/webapp/script.js [hunk 7]
```diff
@@ -15,13 +15,24 @@
...
+       'I enjoy baking',
+        'My favorite book is The Fountainhead by Ayn Rand', 
+        'I am from Guyana',
+        'My favorite food is burgers!',
+        'My current binge is Greys Anatomy'];
 
   // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
-
+  if(count>=greetings.length){
```
Comment (ajw029) [inline portfolio/src/main/webapp/script.js]: I would space this like 
` if(count >= greetings.length) {`

to make it easier to read.

File: portfolio/src/main/webapp/style.css [hunk 8]
```diff
@@ -1,9 +1,37 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+body{
+    color:white;
+    background-color:black;
+}
```
Comment (ajw029) [inline portfolio/src/main/webapp/style.css:5]: A couple of suggestions.

1. After all of your css definitions i would add a new line to make it easier to read. 

Example
```
body {
  color: white;
}

.bleh {
  width: 100%;
}

...
```

2. After each key I would add a space. It can be a bit hard to read when its like `color:white;`

Example
```
  color: white;
  width: 100%;
...
```

3. I would add a space after the class name or id

Example

```
#headerText {

}
```

File: README.md [hunk 9]
```diff
@@ -1,13 +0,0 @@
-# Google Student Training in Engineering Program
```
Comment (tonymai) [inline README.md:1]: You should update the readme instead of deleting it.

https://github.com/bhassan6621/StepTraining/blob/master/walkthroughs/week-1-setup/github-setup-walkthrough.md#modify-readme

This should also be in a separate pull request from updating styling of your portfolio.

File: portfolio/src/main/webapp/index.html [hunk 10]
```diff
@@ -8,11 +8,35 @@
...
+      </div>
+      <div id=" pictures">
+          <!--photo one -->
+            <img width="300px"  height="300px" src="/images/imageOne.jpg"/>
+
+         <!--photo two -->
+            <img width="300px"  height="300px" src="/images/imageTwo.jpg"/>
+
+         <!--photo three -->
+            <img width="300px"  height="300px" src="/images/imageThree.jpg"/>
```
Comment (ajw029) [inline portfolio/src/main/webapp/index.html]: Probably out of scope for this project, but another thing to keep in mind is that this only might not scale well when it comes to mobile form factors. This is usually solved using media queries: https://www.w3schools.com/cssref/css3_pr_mediaquery.asp

Not required to fix for this PR. Just an FYI :)

File: portfolio/src/main/webapp/style.css [hunk 11]
```diff
@@ -1,20 +1,17 @@
-/* #content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
-} */
 
 body{
```
Comment (tonymai) [inline portfolio/src/main/webapp/style.css:2]: You should add a space between the selector and the opening brace.

Take a look at some css style guides:
https://google.github.io/styleguide/htmlcssguide.html#CSS
https://github.com/airbnb/css

File: README.md (hunk 9 above)
Comment (tonymai) [reply README.md:1]: Doesn't look resolved?
